draft-ietf-taps-impl.md

```diff
@@ -150,9 +152,9 @@ It is important to fail as early as possible in such cases in order to avoid all
 
 ## Role of system policy
 
-The properties specified during pre-establishment has a close connection to system policy. The implementation is responsible for combining and reconciling several different sources of preferences when establishing Connections. These include, but are not limited to:
+The properties specified during pre-establishment have a close connection to system policy. The implementation is responsible for combining and reconciling several different sources of preferences when establishing Connections. These include, but are not limited to:
 
-1. Application preferences, i.e., preferences specified during the pre-establishment such as Local Endpoint, Remote Endpoint, Path Selection Properties, and Protocol Selection Properties.
+1. Application preferences, i.e., preferences specified during the pre-establishment via Selection Properties.
```

Comment:
nit: Again mention other properties: "or derived from early specification of Connection or Message Properties"

Reply:
@philsbln The earlier text covers the fact that you may also use connection properties if available. Let us not put them back into the text everywhere again. And there is also a distinction as you must take selection properties into account, but the connection properties are additional input that you can use if you like.